src/errors.ts

```diff
@@ -3,6 +3,9 @@ import { t } from '@lingui/macro'
 export const DEFAULT_ERROR_HEADER = t`Please refresh the page and try again.`
 export const DEFAULT_ERROR_ACTION = t`Reload the page`
 
+const DEFAULT_DISMISSABLE_ERROR_HEADER = t`Please try again.`
+const DEFAULT_DISMISSABLE_ERROR_ACTION = t`Dismiss`
+
 interface WidgetErrorConfig {
```

Comment:
Should this include `dismissable?: boolean`?

Reply:
If you subclass a `DismissableWidgetError`, this should not be exposed in the config. It should auto-populate to `true` if you subclass from `DismissableWidgetError`.